src/main/java/teammates/storage/api/AccountsDb.java

```diff
@@ -238,4 +238,18 @@ private Account getAccountEntity(String googleId) {
     protected Object getEntity(EntityAttributes entity) {
         return getAccountEntity(((AccountAttributes) entity).googleId);
     }
+
+    @Override
+    protected QueryWithParams getEntityKeyOnlyQuery(EntityAttributes attributes) {
+        Class<?> entityClass = Account.class;
+        String primaryKeyName = "googleId";
```

Comment:
Can this be part of the entity class instead? e.g. `public static String PRIMARY_KEY_NAME = "googleId";`